languages/csharp/exercises/concept/casting/.docs/instructions.md

```diff
@@ -0,0 +1,86 @@
+Our football club (first encountered in (cross-ref-tba)) is soaring in the leagues, and you have been invited to do some more work, this time on the laminate printing system.
+
+The class hierarchy of the backroom staff is as follows
+
+```
```

Comment:
I find this class hierarchy quite something. Could we perhaps simplify this a bit?

Reply:
We need the depth to accommodate `is` and  `GetType()`.  We need the width so that the student does not simply say `xxx is Security && !(xxx is SecurityJunior)`.